config.lua

```diff
@@ -7,6 +7,9 @@ local defaultLanguageServerOptions = {
     -- (REQUIRED) command to execute the language server
     cmd = "",
 
+    -- Filetypes recognized by the server. `nil` accepts all filetypes.
+    filetypes = nil,
+
     -- Arguments for the above command
     args = {},
```

Comment:
Probably best to keep `cmd` and `args` next to each other, otherwise the "for the above command" comment is rather confusing.

```suggestion
    -- Arguments for the above command
    args = {},

    -- List of filetypes supported by the server.
    -- Defaults to accepting all filetypes if omitted.
    filetypes = nil,
```